app/viewmodels/govuk/CheckboxFluency.scala

```diff
@@ -74,13 +81,15 @@ trait CheckboxFluency {
                content: Content,
```

Comment:
can we add another apply to have divider with no content  argument, Divider then become string not an option

Reply:
Updated in my next push.